### ISSUE TITLE
loss increasing

### ISSUE THREAD

**Issue by FlyingAnt2018** [2gunsu/monocon-pytorch#24]
Hi, bro. I have just adjusted the epoch from 200 to 300, when training process going to epoch 100, all loss items have a significant increase. Do you know the reason ? Thanks.
![image](https://github.com/2gunsu/monocon-pytorch/assets/49147567/1ab425ce-5cd0-42a9-ae5b-ad350f0cace1)

**Comment by SrinjaySarkar** [comment]
Yeah I have  observed the same exact thing. Could you find any reason @FlyingAnt2018  ?

**Comment by FlyingAnt2018** [comment]
> Yeah I have observed the same exact thing. Could you find any reason @FlyingAnt2018 ?

lr too large

**Comment by SrinjaySarkar** [comment]
Could you provide the the lr you used to train it on 300 epochs ? @FlyingAnt2018

**Comment by 2gunsu** [comment]
Hello. I'm sorry for the delay in answering. 😢
I'll check what you said and get back to you soon.

Thank you.

**Comment by sunhucheng** [comment]
> Hello. I'm sorry for the delay in answering. 😢 I'll check what you said and get back to you soon.
> 
> Thank you.
I have observed the same problem, have you solved it? I think it's a matter of learning rate. @2gunsu

**Comment by sunhucheng** [comment]
I've found that this doesn't happen when the batch size is 8, but it does happen when it's 24. it can resume training from ckpt before loss incressing then loss will decreases.

**Comment by QSQSQSQS** [comment]
I have observed the same exact thing while training with nuscenes, but it didn't happen with KITTI.
